ios/Podfile.lock

```diff
@@ -531,7 +531,7 @@ SPEC CHECKSUMS:
   ReactNativeExceptionHandler: 8025d98049c25f186835a3af732dd7c9974d6dce
   ReactNativeNavigation: aad255980f20cc5d2001ed37972319f3789fea70
   ReactNativePermissions: 2e324bb3baf12702e3ec27f76c8ae1158fa69fc6
-  RealmJS: e3dfc3f31edd9074989e1b6eee811f816afdaea9
+  RealmJS: 8222bc7422d9a7f7a5f95ff43fa838c235d477b9
```

Comment:
Why is this changing?

Reply:
I believe this was my cocoapods version (1.7.5) causing this.